scripts/twitter_post_template.py

```diff
@@ -0,0 +1,17 @@
+from manim import *
+import pkg_resources
+
+version_num = pkg_resources.get_distribution("manimce").version
+
+
+class TwitterScene(Scene):
+    def construct(self):
+        self.camera.background_color = "#ece6e2"
+        version = TextMobject(f"v {version_num}").to_corner(UR).set_color(BLACK)
```

Comment:
I think it would look better without the space between `v` and the version.

```suggestion
        version = TextMobject(f"v{version_num}").to_corner(UR).set_color(BLACK)
```

Reply:
good catch